docs/team/hyngkng.md

```diff
@@ -3,24 +3,27 @@ layout: page
 title: Hwang Yong Kang's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: ProductiveNUS
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+ProductiveNUS is a desktop application targeted at Computing students of National University of Singapore (NUS) to help them manage and schedule their academic tasks efficiently. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to import NUSMods timetable into ProductiveNUS.
+  * What it does: Allows the user to import their NUSMods lesson information into ProductiveNUS.
+  * Justification: This feature improves the product significantly because a user can conveniently refer to their timetable on ProductiveNUS instead of having to got to NUSMods. Other commands such as Schedule also works best if the user can import their weekly timetable.
+  * Highlights: This enhancement retrieves data from the NUSMods API. A new model class Lesson is also implemented to support the import command.
```

Comment:
Typo in the Justification section:
"... instead of having to **got** to NUSMods".